uniro_frontend/src/components/report/secondaryForm.tsx

```
@@ -0,0 +1,51 @@
+							/>
+						);
+					})}
+				{formData.passableStatus === PassableStatus.DANGER &&
+					buttonConfig.danger.map((button, index) => {
+						return (
+							<SecondaryFormButton
+								onClick={handleSecondarySelect}
+								formPassableStatus={formData.passableStatus}
+								isSelected={formData.dangerIssues.includes(button)}
+								contentKey={button}
+							/>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)